Tests/Pcap++Test/Tests/TcpReassemblyTests.cpp

```diff
@@ -120,6 +121,40 @@ static void tcpReassemblyMsgReadyCallback(int8_t sideIndex, const pcpp::TcpStrea
 }
 
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// tcpReassemblyManuallyCloseConnMsgReadyCallback()
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+static void tcpReassemblyManuallyCloseConnMsgReadyCallback(int8_t sideIndex, const pcpp::TcpStreamData& tcpData, void* userCookie)
+{
+	TcpReassemblyMultipleConnStats::Stats &stats = ((TcpReassemblyMultipleConnStats*)userCookie)->stats;
```

Comment:
Nit: c-style cast

Reply:
Updated in this commit: https://github.com/seladb/PcapPlusPlus/pull/1346/commits/1aa0df801960052b3d2acc9b6e4450d16fddb728